packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss

```diff
@@ -105,10 +105,8 @@
       overflow: hidden;
       // positions text and placeholder including when showing placeholder
       // NOTE: Using flex does strange things to a caret in content editable
-      width: calc(100% - var(--#{$block-class}--toolbar-width) - $spacing-05);
-      max-width: calc(
-        100% - var(--#{$block-class}--toolbar-width) - $spacing-05
-      );
+      width: calc(100% - var(--#{$block-class}--toolbar-width) - #{$spacing-05});
+      max-width: calc(100% - var(--#{$block-class}--toolbar-width) - #{$spacing-05});
```

Comment:
Looks like this part is causes the lint error. It should be fixed with the original indentation and spacing though. 🙂

Reply:
Updated spacing :D